packages/create-vite/template-vue-ts/src/components/HelloWorld.vue

```diff
@@ -9,44 +9,32 @@ const count = ref(0)
 <template>
   <h1>{{ msg }}</h1>
 
-  <p>
-    Recommended IDE setup:
-    <a href="https://code.visualstudio.com/" target="_blank">VS Code</a>
-    +
-    <a href="https://github.com/johnsoncodehk/volar" target="_blank">Volar</a>
-  </p>
-
-  <p>See <code>README.md</code> for more information.</p>
+  <div class="card">
+    <button type="button" @click="count++">count is {{ count }}</button>
+    <p>
+      Edit
+      <code>components/HelloWorld.vue</code> to test HMR
+    </p>
+  </div>
 
   <p>
-    <a href="https://vitejs.dev/guide/features.html" target="_blank">
-      Vite Docs
-    </a>
-    |
-    <a href="https://v3.vuejs.org/" target="_blank">Vue 3 Docs</a>
+    Chech out
+    <a
+      href="https://vuejs.org/guide/quick-start.html#with-build-tools"
```

Comment:
nit: I think the landing of this anchor is a bit off...you have to read at least a paragraph to get to where `create-vue` is (or do a search for it). I recommend a closer anchor to `create-vue`: https://vuejs.org/guide/quick-start.html#local